README.md

```diff
@@ -7,3 +7,40 @@
 # WiX Toolset v4
 
 This repository contains the WiX Toolset v4 codebase.
+
```

Comment:
These changes don't go with the title of the PR. Is this about doc or were other files supposed to go in?

Reply:
Lots of other files are going in, this was just a convenient addition (and I was hoping with draft status you wouldn't review, but that's OK)

Reply:
Ahh, my bad. I didn't notice it was still in Draft. I'll watch for that now since it doesn't look like there is anything in the email from GitHub to denote the PR is still in draft.